README.md

```diff
@@ -1 +1,44 @@
-# mobile-me-
\ No newline at end of file
+# mobile-me-
+
+  ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+This is my second attempt for this mobile project mobile portfolio as per the requirements of the project. It is designed using HTML and CSS .
+
+## Built With
+
+html
+css
+
+To get a local copy up and running follow these simple example steps.
+Clone the repo named "portfolio"
+
+## Author
+
+👤 Faishar 
+
+- GitHub: [faishar](https://github.com/Robinson-faike123)
```

Comment:
Great job @Robison-faike123 🤝

- [ ] [Optional] Kindly make sure you add your GitHub handle to your readme file correctly. You may look into [markdown syntax](https://www.markdownguide.org/basic-syntax/) for more guidance.